intent2batch.py

```diff
@@ -99,10 +99,10 @@ def fix_batch_job():
         job_str = json.load(open('./job_to_fix.json'))
```

Comment:
Can we get this from user input, instead of a file? For example you can even live demo how you directly copy the json from a customer issue, and then paste in the tool, which will be cool.

Reply:
Good point, let me try, thanks!